inputs/prod_inner.json

```diff
@@ -48,6 +48,10 @@
       "name": "add_dockerfile"
     },
     {
+      "name": "add_help",
+      "required": false
```

Comment:
Right, this seems a little confusing. Not sure if it would be better to move this plugin before `add_dockerfile` to show this explicitly though